WireUI/Sources/WireConversationUI/ConversationCell/CellTypes/TimeDivider/TimeDividerContentView.swift

```diff
@@ -0,0 +1,128 @@
+//
+// Wire
+// Copyright (C) 2025 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+import WireDesign
+
+struct TimeDividerContentView: ConversationCellContentViewProtocol {
+
+    private let dividerColor = ColorTheme.Strokes.outline.color
+
+    private(set) var model: TimeDividerModel
+
+    var body: some View {
+        HStack(spacing: 0) {
+
+            if model.isUnreadIndicatorVisible {
+                unreadDot
+            } else {
+                leadingDivider
+            }
+
+            if !model.text.isEmpty {
+                text
+            }
+
+            trailingDivider
+
+        }
+        .padding(.vertical, 8)
+    }
+
+    @ViewBuilder private var withoutUnreadIndicator: some View {
```

Comment:
question: is it used?

Reply:
Good catch, it's not used. I moved some code and forgot about it.